trufflehog/templates/deployment.yaml

```diff
@@ -23,15 +23,43 @@ spec:
       {{- else }}
       # no priority class, leave it up to the system
       {{- end }}
+      securityContext:
+        {{- toYaml .Values.securityContext.pod | nindent 8 }}
+      initContainers:
+        - name: copy-binary
```

Comment:
Rather than doing this init container and volume stuff, should we instead modify the permissions where the binary resides so that uid:gid 1000:1000 can modify it?

Reply:
I was actually wondering it that would be cleaner. Let me try that, thanks for the suggestion.

Reply:
https://github.com/trufflesecurity/thog/pull/3524